frontend/app/sysupport/detail/reload/controller.js

```diff
@@ -0,0 +1,35 @@
+import Controller from '@ember/controller'
+import { inject as service } from '@ember/service'
+
+export default Controller.extend({
+  i18n: service(),
+  notify: service(),
+
+  actions: {
+    subscribe(subscriptionPackage) {
+      this._subscribe(subscriptionPackage)
+    }
+  },
```

Comment:
`actions` should always be at the end of the file

Reply:
:+1: